src/renderer/components/TranscriptionBlock.tsx

```diff
@@ -0,0 +1,51 @@
+import styled from '@emotion/styled';
+import { Box, colors } from '@mui/material';
+import { Transcription } from 'sharedTypes';
+
+const TranscriptionBox = styled(Box)`
+  background: ${colors.common.white};
+  height: 100%;
+  padding: 20px;
+`;
+
+const Word = styled('span')`
+  &:hover {
+    background: orange;
```

Comment:
I believe this is supposed to be `yellow-500` with 50% opacity. Sorry, thats my bad, didn't add it on figma in hi-fi designs

Reply:
Yep, styling isn't complete. Just a random color for now.